Godeps/Godeps.json

```diff
@@ -2018,6 +2018,10 @@
 			"Rev": "a0ff2567cfb70903282db057e799fd826784d41d"
 		},
 		{
+			"ImportPath": "github.com/juju/ratelimit",
```

Comment:
surprised this showed up... I thought you repicked the godep... should have been a no-op, right?